ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/helpcenter/articlepage/ArticleFragmentTest.java

```diff
@@ -0,0 +1,514 @@
+package com.kayako.sdk.android.k5.helpcenter.articlepage;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Looper;
+import android.support.design.widget.Snackbar;
+import android.text.format.DateUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.widget.TextView;
+import com.kayako.sdk.android.k5.R;
+import com.kayako.sdk.android.k5.common.utils.ImageUtils;
+import com.kayako.sdk.android.k5.common.utils.ViewUtils;
+import com.kayako.sdk.android.k5.common.view.CircleImageView;
+import com.kayako.sdk.android.k5.common.viewhelpers.DefaultStateViewHelper;
+import com.kayako.sdk.helpcenter.articles.Article;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+import static org.powermock.api.mockito.PowerMockito.doNothing;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.spy;
+import static org.powermock.api.mockito.PowerMockito.verifyStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+import static org.powermock.api.mockito.PowerMockito.whenNew;
+import static org.powermock.api.support.membermodification.MemberModifier.suppress;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+        ArticleFragment.class,
+        Bundle.class,
+        ArticleFactory.class,
+        ImageUtils.class,
+        DateUtils.class,
+        Intent.class,
+        Uri.class,
+        ViewUtils.class,
+        Snackbar.class,
+        Looper.class
+})
+public class ArticleFragmentTest {
+    private static final String ARG_ARTICLE = "article";
+    private static final String NAME = "name";
+    private static final String AVATAR_URL = "testurl";
+    private static final String TITLE = "title";
+    private static final String DIRECTORY_PATH = "path";
```

Comment:
✅ Good to see constant used instead of hard-coded values.

This approach increases maintainability (single point of modification) and debugging, avoid duplication and decreases verbosity.

**No change suggested, looks good!**	  

<a id="7f49f7d0-4e59-11e8-860c-69c5dc5809f4" href="none" codereview-uuid-reserved-tag />